index.bs

```diff
@@ -937,7 +931,24 @@ interface MLGraphBuilder {
 Both {{MLGraphBuilder}}.{{MLGraphBuilder/build()}} and {{MLGraphBuilder}}.{{MLGraphBuilder/buildSync()}} methods compile the graph builder state up to the specified output operands into a compiled graph according to the type of {{MLContext}} that creates it. Since this operation can be costly in some machine configurations, the calling thread of the {{MLGraphBuilder}}.{{MLGraphBuilder/buildSync()}} method must only be a worker thread to avoid potential disruption of the user experience. When the {{[[contextType]]}} of the {{MLContext}} is set to [=default-context|default=], the compiled graph is initialized right before the {{MLGraph}} is returned. This graph initialization stage is important for optimal performance of the subsequent graph executions. See [[#api-mlcommandencoder-graph-initialization]] for more detail.
 </div>
 
-### batchNormalization ### {#api-mlgraphbuilder-batchnorm}
+{{MLGraphBuilder}} has the following internal slots:
+<dl dfn-type=attribute dfn-for="MLGraphBuilder">
+    : <dfn>\[[context]]</dfn> of type {{MLContext}}
+    ::
+        The context of type {{MLContext}} associated with this {{MLGraphBuilder}}.
+</dl>
+
+### The [=new=] {{MLGraphBuilder}} constructor steps are:
```

Comment:
You should remove `###` otherwise bikeshed thinks this line is a section header: https://pr-preview.s3.amazonaws.com/zolkis/webnn/pull/301.html#the-new-mlgraphbuilder-constructor-steps-are

When working with bikeshed I usually build locally to check how the output is rendered. The syntax between bikeshed and ReSpec is quite similar but there are differences such as this one. There's also a bikeshed Web API that works without local bikeshed install: https://github.com/webmachinelearning/webnn/#pull-requests

Reply:
Good catch, will fix all these.